source/dhtslib/vcf/record.d

```diff
@@ -663,6 +941,44 @@ struct VCFRecord
             hts_log_warning(__FUNCTION__, format("Couldn't add tag (ignoring): %s with value %s", tag, data));
     }
 
+    /// returns a hashmap of info data by field
+    InfoField[string] getInfos()
+    {
+        /// unpack
+        if (this.line.max_unpack < UNPACK.INFO) bcf_unpack(this.line, UNPACK.INFO);
+
+        /// copy some data
```

Comment:
Can this be made idiomatic D as you've done here but without copying?

Reply:
Probably.

Reply:
resolved